lib/base64_attach.rb

```diff
@@ -0,0 +1,10 @@
+module Base64Attach
```

Comment:
Just for curiosity, where is this used? :)

Reply:
Nowhere yet, it will be used wherever you want to attach a base64 image, I guess this will be mostly in controllers.

Reply:
The idea was to simplify the attachment of base64 images, allowing to do something like this `user.image.attach(from_base64(params[:user][:image]))`